src/contexts/Api/index.tsx

```diff
@@ -130,6 +131,8 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
 
     const expectedBlockTime = FallbackExpectedBlockTime;
 
+    const expectedEraTime = FallbackExpectedEraTime;
```

Comment:
I needed the era time, so I reckoned I'd get it the same way that the block time is read to stick to the convention - using this `useApi` hook. I haven't found a way to fetch it in the polkadot api docs (like e.g. the bonding duration can be fetched a couple lines above), so I just hardcoded it like `expectedBlockTime` is.

Reply:
Good, so seems I can leave it this way for now and we adapt to api calls when needed 👍